### PR TITLE
IGVF-1048-linking-on-supplement

### PR DESCRIPTION
**Pull request by ian-whaling**

### CODE REVIEW

File: components/search/list-renderer/analysis-set.js [hunk 1]
```diff
@@ -20,7 +21,14 @@ export default function AnalysisSet({ item: analysisSet }) {
...
     analysisSet.input_file_sets?.length > 0
-      ? analysisSet.input_file_sets.map((fileSet) => fileSet.accession)
+      ? analysisSet.input_file_sets
+          .map((fileSet, index) => [
+            <Link key={fileSet["@id"]} href={fileSet["@id"]}>
+              {fileSet.accession}
+            </Link>,
+            index < analysisSet.input_file_sets.length - 1 && ", ",
+          ])
+          .flat()
```
Comment (forresttanaka) [inline components/search/list-renderer/analysis-set.js:31]: You can do this in an easier and more standardized way using the [SeparatedList component](https://github.com/IGVF-DACC/igvf-ui/blob/dev/components/separated-list.js). We have examples all over the code base. You can just collect up all the input file sets and flatten them into one array, then put a map loop within `<SeparatedList>`. Probably easier to do the render part inline with `<SearchListItemSupplementContent>`.

File: components/search/list-renderer/file.js [hunk 2]
```diff
@@ -26,6 +27,17 @@ export default function File({ item: file, accessoryData = null }) {
...
+    file.seqspec_of?.length > 0 && accessoryData
+      ? file.seqspec_of
+          .map((key, index) => [
+            <Link key={key} href={key}>
+              {accessoryData[key].accession}
+            </Link>,
+            index < file.seqspec_of.length - 1 && ", ",
+          ])
+          .flat()
+      : [];
```
Comment (forresttanaka) [inline components/search/list-renderer/file.js:40]: Same thing as the analysis-set.js file, using `<SeparatedList>`.

File: components/search/list-renderer/file.js [hunk 3]
```diff
@@ -48,16 +60,28 @@ export default function File({ item: file, accessoryData = null }) {
...
+                </SearchListItemSupplementLabel>
+                <SearchListItemSupplementContent>
+                  <Link href={fileSet["@id"]}>{fileSet.summary}</Link>
+                </SearchListItemSupplementContent>
+              </SearchListItemSupplementSection>
+            )}
+            {seqspecOfFileLinks.length > 0 && (
+              <SearchListItemSupplementSection>
+                <SearchListItemSupplementLabel>
+                  Seqspec of
```
Comment (forresttanaka) [inline components/search/list-renderer/file.js:78]: Technically a capital on the `of`, though I think this is styled as all uppercase anyway.

**Review by forresttanaka — CHANGES_REQUESTED** [review]
Nice! Nothing wrong but I have a suggestion for a tool we already have to make the code easier and more easily recognizable for what it does at a glance.

File: components/search/list-renderer/analysis-set.js (hunk 1 above)
Comment (ian-whaling) [reply components/search/list-renderer/analysis-set.js]: Replaced.

File: components/search/list-renderer/file.js (hunk 2 above)
Comment (ian-whaling) [reply components/search/list-renderer/file.js]: Replaced.

File: components/search/list-renderer/file.js (hunk 3 above)
Comment (ian-whaling) [reply components/search/list-renderer/file.js]: Replaced.

**Pull request merged**